src/contentApi/cost-api.js

```diff
@@ -43,64 +43,71 @@ const createCostSourcesLink = (type) =>
     getSourceTypesIDs()[type]
   }&filter[applications][application_type_id][eq][]=${getCostAppId()}`;
 
-export const createCostSchema = () => ({
-  firstPanel: [
+export const createCostSchema = async () => {
+  const params = ['OCP', 'AWS', 'Azure', 'GCP'];
+  const linkParams = ['openshift', 'amazon', 'azure', 'google'];
+  const [OCP, AWS, Azure, GCP] = await Promise.all(
+    params.map((param) => costRequest(param))
+  );
+  const [openshiftLink, amazonLink, azureLink, googleLink] = await Promise.all(
+    linkParams.map((param) => createCostSourcesLink(param))
```

Comment:
in this request there is a method to get cost app id

This ID should be obtained before to reduce the number of calls and passed to this function as a second argument